### DISCUSSION TITLE
[Vuetify v3.0.0-alpha.12] Component Testing with cypress

### DISCUSSION THREAD

**Discussion by GregKukanich** [vuetifyjs/vuetify#14454] [Q&A]
Currently working using Vue 3 + Pinia +  Vuetify 3 + Vite  + Cypress. I am trying to test one of my components using cypress. In this component I use the <v-app-bar> component from the Vuetify framework. When I run the cypress test I get the following error before the test is able to be run, "[Vuetify] Could not find defaults instance" .  Just wondering if anyone could offer some help in resolving this problem? Thanks!

Below is the code for my test.

```TypeScript
import { mount } from '@cypress/vue'
import { setActivePinia, createPinia } from 'pinia'
import MainView from "@/views/MainView.vue";

describe('MainView', () =>{
    beforeEach(() => {
        setActivePinia(createPinia())
    })
    it('testing main view', () => {
        mount(MainView)
        cy.get("v-app-bar").should("exist")
    })
})
```

**Comment by muassab** [comment]
I resolved this problem by adding `createVuetify` in `beforeEach` inside of `describe`:

```javascript
describe('should ...', () => {
  beforeEach(() => {
    const vuetify = createVuetify();
    cy.mount(MyComponent, {
      global: { plugins: [vuetify] }
    });
  });

  it('test', () => {
     // ...
  });
```
